### PR TITLE
fix: hide script header

### PR DESCRIPTION
**Pull request by ctizen**

### CODE REVIEW

**Review by spalt08 — APPROVED** [review]

File: packages/web/src/components/ui/PackagePreview/PackagePreview.tsx [hunk 1]
```diff
@@ -82,27 +82,29 @@ export default function PackagePreview({
       >
         <div className={styles.content}>
           <div className={styles.contentInner}>
-            <div className={styles.stat}>
-              <div className={styles.statHeader}>
-                <Icon kind='script' color='#8E8AA0' className={styles.statIcon} />
-                Script
+            {(pkg.containingScripts?.length ?? 0) > 0 && (
```
Comment (zardak) [inline packages/web/src/components/ui/PackagePreview/PackagePreview.tsx:85]: nit: We're usually sticking to this pattern across the codebase `!!pkg.containingScripts?.length`

**Review by zardak — APPROVED** [review]

File: packages/web/src/components/ui/PackagePreview/PackagePreview.tsx [hunk 2]
```diff
@@ -82,27 +82,29 @@ export default function PackagePreview({
...
+              <div className={styles.stat}>
+                <div className={styles.statHeader}>
+                  <Icon kind='script' color='#8E8AA0' className={styles.statIcon} />
+                  Script
+                </div>
+                {detailsLoading ? (
+                  <ScriptSkeleton />
```
Comment (zardak) [inline packages/web/src/components/ui/PackagePreview/PackagePreview.tsx:92]: Also, seems like `<ScriptSkeleton>` will never be shown with current logic and data flow

File: packages/web/src/components/ui/PackagePreview/PackagePreview.tsx (hunk 1 above)
Comment (spalt08) [reply packages/web/src/components/ui/PackagePreview/PackagePreview.tsx:85]: It should also work without `!!`

File: packages/web/src/components/ui/PackagePreview/PackagePreview.tsx (hunk 2 above)
Comment (ctizen) [reply packages/web/src/components/ui/PackagePreview/PackagePreview.tsx:92]: True. I suggest dealing with it a bit later though.

**Pull request merged**